ingenialink/canopen/network.py

```diff
@@ -573,7 +579,10 @@ def __program_control_to_stop(
         logger.info("Flashing firmware")
         with contextlib.suppress(ILError):
             servo.write(PROG_STAT_1, PROG_CTRL_STATE_STOP, subnode=0)
-        servo.node.nmt.start_node_guarding(CANOPEN_BOTT_NODE_GUARDING_PERIOD)
+        try:
+            servo.node.nmt.start_node_guarding(CANOPEN_BOTT_NODE_GUARDING_PERIOD)
+        except VCIError as e:
+            raise ILFirmwareLoadError(f"Firmware loading error using ixxat device") from e
```

Comment:
I think that the message should not mention the IXXAT brand. Maybe something like: "An error occurred when starting the node guarding."

Reply:
Ok, I'm going to add a comment to let developers know that it's a specific error for ixxat transceivers.